lib/pacemaker/pcmk_sched_colocation.c

```diff
@@ -1073,26 +1073,6 @@ pcmk__apply_coloc_to_priority(pe_resource_t *dependent, pe_resource_t *primary,
                                            dependent->priority);
 }
 
-/*!
- * \internal
- * \brief Apply a colocation constraint to allowed nodes' weights
- *
- * \param[in] colocation   Colocation constraint to apply
- * \param[in] rsc1         Resource whose allowed nodes should be updated
- * \param[in] rsc2         Resource whose preferences should be added
- * \param[in] flags        Flags (enum pe_weights) to apply when adding scores
- */
-void
-pcmk__apply_colocation(pcmk__colocation_t *colocation, pe_resource_t *rsc1,
```

Comment:
Commit message for this commit (72a2f8e): "This was a convenience wrapper added by 05f40aa6cf" <- That commit isn't showing up for me. Maybe 514969ca?

Reply:
Whoops, yes, that was from an earlier checkout that got rebased